vidarr-server/src/main/java/ca/on/oicr/gsi/vidarr/server/DatabaseBackedProcessor.java

```diff
@@ -190,13 +191,13 @@ static String computeWorkflowRunHashId(
       Iterable<String> labels,
       TreeSet<String>
           inputIds, // In both existing calls, hashFromAnalysisId has already been called. TODO:
-                    // maybe refactor if any more calls are needed
+      // maybe refactor if any more calls are needed
       Collection<? extends ExternalId> externalIds) {
     try {
       final var digest = MessageDigest.getInstance("SHA-256");
       digest.update(name.getBytes(StandardCharsets.UTF_8));
       for (final var id : inputIds) {
-        final var idBytes = id.getBytes(StandardCharsets.UTF_8);
+        final var idBytes = hashFromAnalysisId(id).getBytes(StandardCharsets.UTF_8);
```

Comment:
Same comment as #122

Reply:
If you need to make this change, you'll need to remove the calls to hashFromAnalysisId present before each place this method is called. Please then remove the comment on lines 192-193

Reply:
I've made this change because I figure that federation between instances means that it would be useful to set a convention to pass around the full Vidarr identifier and only extract specific portions (e.g. hash) where it is specifically needed.